### ISSUE TITLE
perichondritis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11636]
**Preferred term label:**
perichondritis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
